components/expenses/PayoutMethodForm.js

```diff
@@ -33,7 +33,7 @@ const msg = defineMessages({
 });
 
 /** Use this function to validate the payout method */
-export const validatePayoutMethod = payoutMethod => {
+export const validatePayoutMethod = (payoutMethod, legalName) => {
```

Comment:
It looks like this function is used in other files that haven't been updated (esp. `components/expenses/ExpenseForm.js`). Is it fine?